app/assets/javascripts/questions.js

```diff
@@ -74,10 +95,16 @@ function enableSubmitQuestion() {
   }
 }
 
+// allows the function to be enabled on the generate-question page 
```

Comment:
This is a bit vague -- what is 'the function'? Maybe more specificity, like:
```
// enables/disables the submit question button on page load
```
Note: questions.js could technically be brought in on other pages, so specifically calling out the generate-question page might be confusing if this is called from elsewhere.